core/src/main/java/org/neo4j/ogm/context/GraphEntityMapper.java

```diff
@@ -161,10 +161,10 @@ private void mapNodes(GraphModel graphModel, Set<Long> nodeIds) {
 						synchronized (this) {
 							entity = entityFactory.newObject(node);
 							setIdentity(entity, node.getId());
+							setProperties(node, entity);
 							entity = mappingContext.registerNodeEntity(entity, node.getId());
```

Comment:
The reason setProperties is called whether or not the the node entity is in the mapping context is because when you reload the entity from the graph, we always update properties. This fix indicates that it will no longer be the case. Unfortunately we appear to be missing a test for this, will push one shortly to demo the issue